src/loaders/lottie/tvgLottieAnimation.cpp

```diff
@@ -124,6 +124,17 @@ Result LottieAnimation::assign(const char* layer, uint32_t ix, const char* var,
 }
 
 
+Result LottieAnimation::resolve(std::function<bool(Paint* paint, const char* src, void* data)> callback, void* data) noexcept
+{
+    delete PICTURE(pImpl->picture)->assetResolver;
+    PICTURE(pImpl->picture)->assetResolver = new AssetResolver{callback, data};
+
+    auto loader = PICTURE(pImpl->picture)->loader;
+    if (loader) static_cast<LottieLoader*>(loader)->resolve(PICTURE(pImpl->picture)->assetResolver);
```

Comment:
I guess you can try this:
`if (loader) static_cast<LottieLoader*>(loader)->resolver(cb, data);`